appservice/src/deploy/deploy.ts

```diff
@@ -80,9 +81,17 @@ export async function deploy(client: SiteClient, fsPath: string, context: IDeplo
                 if (!(await fse.pathExists(fsPath))) {
                     throw new Error(localize('pathNotExist', 'Failed to deploy path that does not exist: {0}', fsPath));
                 }
-
-                if (config.linuxFxVersion && /^(tomcat|wildfly)/i.test(config.linuxFxVersion)) {
+                const javaRutime = client.isLinux ? config.linuxFxVersion : config.javaContainer;
+                if (javaRutime && /^(tomcat|wildfly|jobss)/i.test(javaRutime)) {
```

Comment:
Isn't it supposed to be `jboss`? https://www.redhat.com/en/technologies/jboss-middleware/application-platform

Reply:
Yes, its a typo, thanks a lot!